### DISCUSSION TITLE
실제 유저들한테 받을 피드백 질문지

### DISCUSSION THREAD

**Discussion by mcodnjs** [woowacourse-teams/2023-hang-log#637] [토론]
### 행록 피드백 질문지 
행록 홍보를 하면서 받을 _**피드백 질문지에 어떤 컨텐츠가 들어가면 좋을지**_ !! 고민해봅시다!!
질문지와 함께 사용자한테 입력 받을 형식(ex. 객관식, 주관식) 도 같이 적어주시면 좋겠습니다!

ex) 예시 예시 예시입니다. 예시는 예시일뿐입니다. 
- 행록이 당신의 여행 기록에 도움이 되었나요? (객관식, 점수 선택 1-5) 
- 사용을 하면서 불편했던 점이 있나요? 있다면, 어떤 부분이었는지 적어주시면 감사하겠습니다 (주관식) 
- 행록에서 더 필요하다고 생각한 기능은 무엇인가요? (주관식)

**Comment by jjongwa** [comment]
1. 우리 서비스를 어디에서 처음 접했는지 - (인스타, 카페, 오픈카톡, 기타) -> GA로 대체 가능할지도?
2. 회원가입 했는지? - Y or N
3. 하지 않았다면 왜 하지 않았는지 - (객관식으로 하면 좋을 듯)
4. 행록에서 가장 인상 깊었던 점은 무엇인지 - 중복 가능 (디자인, 가계부, 장소 등록, 카테고리 설정, 사진 등등)
5. 아쉬웠던 부분 or 추가되었으면 하는 사항

전체적으로 주관식보단 객관식으로 체크할 수 있는 질문들 위주로 만들어야 사용자들이 귀찮아하지 않을 것 같습니다..!
질문의 수도 최대한 줄여야 할 듯.?

**Comment by LJW25** [comment]
- 행록을 다시 사용할 생각이 있으신가요? (객관식, 점수 선택) + 이유는 무엇인가요?(주관식, 선택)
- 행록을 다른 사람에게 추천해주고 싶은 생각이 있으신가요? (객관식, 점수 선택) + 이유는 무엇인가요?(주관식, 선택)
- 자유롭게 의견을 남겨주세요 (주관식,선택)

개인적으로 질문을 짧고 단순하게 가져가는게 응답자 입장에서 부담이 덜하지 않을까 하는 생각?!
물론 성의 있는 질문으로,,

**Comment by mcodnjs** [comment]
- 행록을 사용하면서 가장 좋았던 기능은 무엇이었나요? (객관식, 여행 기록, 여행 루트, 공유, 가계부 등)
- 행록 서비스를 다른 사람에게 추천한다면, 어떤 이유로 추천을 하고 싶나요? (객관식)
- 여행을 기록할 때, 행록을 다시 사용할거 같나요 ? (객관식 - 아주 그렇다, 그렇다, 보통이다, 등등)
- 행록 서비스를 개선하기 위한 제안이나 아이디어, 개선점 등이 있으면 남겨주세요. (주관식)
  - 개선점이라면, 토글로 기능 선택(ex. UI/UX, 로딩 시간 등)하고 옆에 추가 설명 적게 할 수 있게??? (할수잇나)

저도 저도 피드백하는 사용자들이 귀찮지 않게 ~ 
핵심적인 질문들만 뽑아서 간략하게 하는거 좋아요!

**Comment by hgo641** [comment]
* 행록에 접속하게 된 계기가 무엇인가요?
  * 다녀온 여행을 기록하기 위해
  * 다른 사람의 여행을 재미로 구경하기 위해
  * 다른 사람의 여행을 토대로 내 여행을 계획하기 위해
  * 그 외(서술형)
* 행록에서 여행을 만들어 보셨나요? (예스인 사람들만)
  * 여행을 만드는 데 어려움을 겪으셨나요?
    * 여행 생성 방식이 복잡함
    * 입력해야 하는 정보가 많아서 번거로움
    * 장소를 검색하기 힘듦
    * 서술형
* 지도에서 루트를 보여주는 기능이 도움이 되었나요? (네/아니오)

**Comment by waterricecake** [comment]
- 행록을 접하게 된 경로는 무엇인가요? (오픈카카오톡, 지인, 인스타, 블로그, 기타(이 경우 누구에게 받았는 지 받아봄))
- 만드신 여행을 지인들에게 공유해 보셨나여?
    - yes -> 공유하신 이유는 무엇인가요?(여행 정보 공유, 자랑하기 등)
        - 공유를 하시면서 불편하셨던 점이 있으신가요?
    - no -> 공유를 하지 않으신 이유는 무엇인가요?
- 만드신 여행을 전체공개 하셨나요? 
    -  yes 
        - 공개를 하신 이유는 무엇인가요?
        - 공개를 하시면서 불편하셨던 점이 있으신가요?
   - no
        - 공개를 하지 않은 이유는 무엇인가요?
       
저는 이번에 추가되는 기능을 사용했는지 또 사용하지 않았으면 그 이유를 물어보는게 좋지 않을까 하여 관련된 질문을 넣었습니다.

**Comment by Dahyeeee** [comment]
평소 여행을 기록하시나요?
기록한다면 어디에 기록하시나요? 
행록에 여행을 기록해보셨나요?
(했다면) 행록이 여행 기록에 도움이 되었나요? (0-10)

다른 질문들은 여러분이 잘해주셔서 저는 새로운 질문을 넣어봤습니다. 호호
